### PR TITLE
Use api to get Playground ID instead of ID env var. #392.

### LINKED ISSUE #438: For env_vars.js, create a function that gets all the environment variables
I swear there was an issue for this, but I can't find it right now.

Instead of having `env_vars.js` create the objects `env_vars` once, make a function that processes and gets environment-variable-based values fresh each time, then use that function everywhere we currently need to get .

We still need to find a different name for this. Any ideas?

### PR DESCRIPTION
**Pull request by plocket**
Addresses #392, closes #438. [As part of this change, env_vars is no longer needed, so it's deleted, and checks for `session_vars` props needed to be moved around as this gets the ID in a different way.]

### CODE REVIEW

File: lib/docassemble/get_base_interview_url.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  /** Return the part of the interview URL that comes before the filename.
+  *    Depends on the project name, which may get changed.
+  *    This also needs to ensure that variable values from previous tests aren't used.
+  *    That's not completely possible, but our tests try to do this in several ways.
+  *    This function uses `new_session=1` to try to ensure old session vars aren't used.
+  *    When finishing a test, we go to the sign out page (which is supposed to help even
+  *    when the user is not logged in.) We can't use `reset=2` because it caused an
+  *    'Unable to locate interview session' error. */
+  let server_url = session_vars.get_da_server_url();
+  let dev_id = await docassemble_api_interface.get_dev_id();  // This seems like a strange place for this
```
Comment (BryceStevenWilley) [inline lib/docassemble/get_base_interview_url.js]:
```suggestion
  let dev_id = await docassemble_api_interface.get_dev_id();
```

File: lib/utils/session_vars.js [hunk 2]
```diff
@@ -186,3 +116,46 @@ session_vars.delete_project_name = function () {
...
+  }
+  if ( !session_vars.get_da_server_url() )
+  {
+    errorMessage += `\nThe SERVER_URL GitHub secret must be defined. The SERVER_URL is the URL of the Docassemble Server where the tests should be run.`;
+  }
+  if ( !session_vars.get_repo_url() )
+  {
+    errorMessage += `\nThe REPO_URL GitHub secret must be defined. The REPO_URL is the URL to the Git repo whose Actions are being run.`;
+  }
+  if ( !session_vars.get_branch_name() )
```
Comment (BryceStevenWilley) [inline lib/utils/session_vars.js]:
```suggestion
  }
  // The branch name is always automatically defined in Github, it'll be missing only if you 're running locally and missing an env var.
  if ( !session_vars.get_branch_name() )
```

**Comment by plocket** [comment]
Just fyi, this one needs an approving review to be merged @BryceStevenWilley

File: lib/utils/session_vars.js [hunk 3]
```diff
@@ -186,3 +116,47 @@ session_vars.delete_project_name = function () {
...
+  }
+  if ( !session_vars.get_da_server_url() )
+  {
+    errorMessage += `\nThe SERVER_URL GitHub secret must be defined. The SERVER_URL is the URL of the Docassemble Server where the tests should be run.`;
+  }
+  if ( !session_vars.get_repo_url() )
+  {
+    errorMessage += `\nThe REPO_URL GitHub secret must be defined. The REPO_URL is the URL to the Git repo whose Actions are being run.`;
+  }
+  // The branch name is always automatically defined in Github, it'll be missing only if you 're running locally and missing an env var.
```
Comment (BryceStevenWilley) [inline lib/utils/session_vars.js]:
```suggestion
  // The branch name is always automatically defined in Github, it'll be missing only if you're running locally and missing an env var.
```

**Review by BryceStevenWilley — APPROVED** [review]
apologies for the late approval!

**Pull request merged**